test/TestProjects/MgmtSafeFlatten/Generated/Models/UnknownLayerOneBaseType.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace MgmtSafeFlatten.Models
+{
+    /// <summary> The UnknownLayerOneBaseType. </summary>
+    internal partial class UnknownLayerOneBaseType : LayerOneBaseType
+    {
+        /// <summary> Initializes a new instance of UnknownLayerOneBaseType. </summary>
+        public UnknownLayerOneBaseType()
```

Comment:
this ctor should not be needed as it will never be called

Reply:
Removed